packages/components/src/DataTable/Body.tsx

```diff
@@ -45,5 +54,12 @@ export function Body<T extends object>({ table, onRowClick }: BodyProps<T>) {
         );
       })}
     </tbody>
+  ) : (
+    <div className={classNames(styles.emptyState)} style={{ height }}>
+      <div className={classNames(styles.emptyMessage)}>
+        <Heading level={5}>No items found</Heading>
```

Comment:
This heading should be removed as well, and I think the divs around it are no longer needed? I don't think there should be any styling or UI elements in here that consumers would have to override to achieve their intended design. I can see maybe keeping the `emptyState` div on line 58 remaining as a generic "we can assume users will want to drop something in the middle" container that's easy enough to do custom work within, with a few styling tweaks.